vulture_os/toolkit/api_parser/trendmicro_visionone/trendmicro_visionone.py

```diff
@@ -231,6 +236,7 @@ def execute(self):
                     f"[{__parser__}]:execute: Log collection for {kind}'s logs stopped at {getattr(self.frontend, f'trendmicro_visionone_{kind}_timestamp')}",
                     extra={'frontend': str(self.frontend)})
 
+        self.frontend.save()
```

Comment:
Introduced in https://github.com/VultureProject/vulture-gui/pull/539, we don't save all the frontend anymore.

```suggestion
        self.frontend.save(update_fields=['trendmicro_visionone_alerts_timestamp', 'trendmicro_visionone_audit_timestamp', 'trendmicro_visionone_oat_timestamp'])
```